pkg/apis/sealed-secrets/v1alpha1/sealedsecret_expansion.go

```diff
@@ -13,13 +13,17 @@ import (
 	"github.com/bitnami-labs/sealed-secrets/pkg/crypto"
 )
 
-func labelFor(o metav1.Object) ([]byte, bool) {
-	label := o.GetAnnotations()[SealedSecretClusterWideAnnotation]
-	if label == "true" {
-		return []byte(""), true
+// Returns labels followed by clusterWide followed by namespaceWide.
+func labelFor(o metav1.Object) ([]byte, bool, bool) {
+	clusterWide := o.GetAnnotations()[SealedSecretClusterWideAnnotation]
+	if clusterWide == "true" {
+		return []byte(""), true, false
 	}
-	label = fmt.Sprintf("%s/%s", o.GetNamespace(), o.GetName())
-	return []byte(label), false
+	namespaceWide := o.GetAnnotations()[SealedSecretNamespaceWideAnnotation]
+	if namespaceWide == "true" {
+		return []byte(fmt.Sprintf("%s", o.GetNamespace())), false, true
```

Comment:
Micro-optimisation: We don't need to copy the namespace string here:

```suggestion
		return []byte(o.GetNamespace()), false, true
```